src/amp-story-player/amp-story-player-impl.js

```diff
@@ -256,14 +256,18 @@ export class AmpStoryPlayer {
   initializeShadowRoot_() {
     this.rootEl_ = this.doc_.createElement('main');
 
-    // Create shadow root
-    const shadowRoot = this.element_.attachShadow({mode: 'open'});
+    // TODO(): Update unit tests to work without shadow root,
+    // then update condition to getMode.test and update visual tests.
+    const containerToUse =
```

Comment:
Can we use this helper instead? It should do everything you need:

https://github.com/ampproject/amphtml/blob/cf610829b0ac7b1d0a07b6aafedf16aa08e05cfe/extensions/amp-story/1.0/utils.js#L94-L111

Reply:
We can't because the unit tests don't work without shadow root.

Reply:
Can you try changing the tests to use `getShadowRootForTesting()` as used here?

https://github.com/ampproject/amphtml/blob/69e8082c05c6704f9cb1b2428ff9c25b27c809c2/extensions/amp-story/1.0/test/test-amp-story-embedded-component.js#L280-L282

here's how we use this helper in another part of the codebase
https://github.com/ampproject/amphtml/blob/69e8082c05c6704f9cb1b2428ff9c25b27c809c2/extensions/amp-story/1.0/amp-story-embedded-component.js#L707